.doctrine-project.json

```diff
@@ -11,21 +11,27 @@
             "upcoming": true
         },
         {
-            "name": "2.1",
-            "branchName": "2.1.x",
-            "slug": "2.1",
+            "name": "2.2",
+            "branchName": "2.2.x",
+            "slug": "latest",
```

Comment:
afaik `slug` has to be unique.

Reply:
Also, it's idiotic to use latest twice :stuck_out_tongue: